Epam.TestAutomation/Epam.TestAutomation.BDD/DataDrivenTestingTests/TestMethods.cs

```diff
@@ -0,0 +1,123 @@
+﻿using Epam.TestAutomation.Core.Browser;
+using Epam.TestAutomation.Helper;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Epam.TestAutomation.Utilities.Logger;
+using OpenQA.Selenium;
+using TechTalk.SpecFlow.BindingSkeletons;
+
+namespace Epam.TestAutomation.BDD.DataDrivenTestingTests
+
+{
+    public static class TestMethods
```

Comment:
The same comments as for Page Object pull request. 
Create the appropriate classes for elements. And move all steps into test.